src/app/organization/tellers/tellers.component.html

```diff
@@ -54,7 +54,7 @@
       </ng-container>
```

Comment:
The `View Cashiers` button is also directing me to the view teller page. Make the necessary changes to ensure that clicking on it does not redirect me to the view teller page.

Reply:
When `View Cashiers` component will be added, the `routerLink` will automatically change to redirect to that component. Since it has not been added yet, everything on the row will redirect to the View Teller component.

Reply:
I have added a `stopPropagation` event to the `View Cashiers` button to temporarily disable the `routerLink` until the View Cashiers component is added.